lib/computer.ex

```diff
@@ -2,10 +2,26 @@ defmodule Computer do
   defstruct mark: "O"
 
   def move(game) do
+    game_type = if game.board |> length < 9, do: "3x3", else: "4x4"
+    available_spaces_number = game.board |> Board.available_spaces_number
+    pick_move(game_type, game, available_spaces_number)
+  end
+
+  def pick_move("3x3", game, available_spaces_number) when available_spaces_number >=7 do
+    move = Enum.random(0..8)
+    if Board.is_available?(game.board, move), do: Game.mark_spot(game, move), else: move(game)      
+  end
+
+  def pick_move("4x4", game, available_spaces_number) when available_spaces_number >= 13 do
+    move = Enum.random(0..15)
+    if Board.is_available?(game.board, move), do: Game.mark_spot(game, move), else: move(game)
```

Comment:
Like the `create` function above, the only thing that's different about these two functions is the size of the range. You could refactor this one as well into one function that takes a board size instead of a string.